app/assets/javascripts/deadline_day_pickers.js

```diff
@@ -0,0 +1,72 @@
+$(document).ready(function () {
```

Comment:
Can you add some screenshots / description of how this affects the entry pages?

Reply:
It provides sort of a live validation and projection as to what will happen when:

![image](https://user-images.githubusercontent.com/22677/151542388-58285936-9688-42bf-b35e-9b6bfbd8ce41.png)
![image](https://user-images.githubusercontent.com/22677/151542394-e5cbc7e4-2950-471a-a22b-b4b71d4c116b.png)
![image](https://user-images.githubusercontent.com/22677/151542405-1e0b1ff9-8e36-44f3-a86c-7b0d7b0b7d9e.png)